test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -167,6 +167,19 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task Page_Route_RouteDataCoercion()
```

Comment:
Not sure what this test method name means. What gets coerced with what? I prefer when test names are fairly specific about what they expect...